.github/workflows/maven.yml

```diff
@@ -22,11 +22,11 @@ jobs:
         with:
           java-version: '11'
           distribution: 'adopt'
-          cache: gradle
+          cache: maven
       - name: Set up OpenPolicyAgent
         run: |
-          curl -L -o opa https://openpolicyagent.org/downloads/v0.33.1/opa_linux_amd64_static
+          curl -L -o opa https://openpolicyagent.org/downloads/v0.46.1/opa_linux_amd64_static
```

Comment:
Now that i see this. We only support OPA up to 0.45.0 i think. Eventually we should check against versions we support?